src/DimpleB/StringArrayProblem15.java

```diff
@@ -0,0 +1,39 @@
+
+/* 15)	WAP to find string which has length more than 7 characheter.
+a.	Input array ={�krishna�, �maulik�, �aakash�, �harsh�, �aakansha�, �abhishek�}
+b.	Output :- Krishna, aakansha, abhishek
+*/
+import java.util.Scanner;
+public class StringArrayProblem15 {
+	
+	void printgivenlenstr(String inputstr[],int strlen)
```

Comment:
Hi @DimpleBAug19 
Here no need to give second param.. it is size of array.. we cal calculate the using .length property in array..
please correct that change in next PR